org.adempiere.base/src/org/compiere/util/DisplayType.java

```diff
@@ -174,6 +175,10 @@
 	public static final int ChosenMultipleSelectionTable = REFERENCE_DATATYPE_CHOSEN_MULTIPLE_SELECTION_TABLE;
 	
 	public static final int ChosenMultipleSelectionSearch = REFERENCE_DATATYPE_CHOSEN_MULTIPLE_SELECTION_SEARCH;
+	
+	public static final int RelatedTo = REFERENCE_DATATYPE_RELATED_TO;
```

Comment:
hi @PeterTakacs300 , the rename of RelatedTo to RecordId is incomplete.

Reply:
Thank you, renaming completed.